application/models/Verification_Model.php

```diff
@@ -108,6 +108,44 @@ public function hasUserPaidEvent($emailAddress, $sheetName)
     }
 
 
+
+    /**
+     * Get user's full name, UPI and UID on the membership spreadsheet.
+     *
+     * @param string $emailAddress The email of the user.
+     *
+     * @return array [full_name, UPI, UID]
+     */
+    public function getUserInfo($emailAddress)
+    {
+        $this->load->model("GoogleSheets_Model");
+
+        if (!$this->isEmailOnSheet($emailAddress, MEMBERSHIP_SPREADSHEET_ID, MEMBERSHIP_SHEET_NAME)) {
+            return false;
+        }
+
+         // Given that the email exists in the sheet, find its index
+        $emailKey = array_search($emailAddress, $this->addresses);
+
+        // Convert key to google coordinate
+        $nameIndex = 'C' . ($emailKey+2);
+        $upiIndex = 'H' . ($emailKey+2);
+        $uidIndex = 'G' . ($emailKey+2);
+
+
+        // Get user's full name
+        $userFullName = $this->GoogleSheets_Model->getCellContents($nameIndex, $nameIndex)[0][0];
+
+        // Get user's UPI
+        $userUpi = $this->GoogleSheets_Model->getCellContents($upiIndex, $upiIndex)[0][0] == null ? 'N/A' : $this->GoogleSheets_Model->getCellContents($upiIndex, $upiIndex)[0][0];
+
+        // Get user's UID
+        $userUid = $this->GoogleSheets_Model->getCellContents($uidIndex, $uidIndex)[0][0] == null ? 'N/A' : $this->GoogleSheets_Model->getCellContents($uidIndex, $uidIndex)[0][0];
+
+        return [$userFullName, $userUpi, $userUid];
```

Comment:
Remove UID references, and don't put a space at the end of the function block

i.e.

```
public function foo() {
    return [...];
    // No line break on this line
}